test/metabase/api/session_test.clj

```diff
@@ -349,6 +349,28 @@
         (is (= {:valid false}
                (mt/client :get 200 "session/password_reset_token_valid", :token token)))))))
 
+(deftest reset-token-ttl-hours-test
+  (testing "Test reset-token-ttl-hours-test"
+    (testing "reset-token-ttl-hours-test is reset to default when not set"
+      (mt/with-temp-env-var-value [mb-reset-token-ttl-hours nil]
+                                  (is (= 48 (setting/get-value-of-type :integer :reset-token-ttl-hours)))))
```

Comment:
These should also be indented just two spaces, per codebase convention

Reply:
changed